src/components/Section/Section.css

```diff
@@ -25,7 +25,7 @@
   letter-spacing: 0em;
   text-align: left;
   margin: 0;
-  padding: 0;
+  padding: 0 0 70px 0;
```

Comment:
Is the last 0 redundant since the second will apply to top & bottom if you don't specify?

Reply:
Yes that's true, I changed it